ExonCov/forms.py

```
@@ -40,7 +40,10 @@ def get_gene(gene_id):
                 ', '.join([gene_alias.gene_id for gene_alias in gene_aliases])
             )
         else:
-            error = 'Unknown gene: {0}.'.format(gene_id)
+            try:
+                error = 'Unknown gene: {0}.'.format(gene_id)
+            except UnicodeEncodeError:
+                error = 'Unparsable gene in list, please remove special characters (e.g. alpha or beta).'
```

maybe add another except?
something like
```
try:
  print(x)
except NameError:
  print("Variable x is not defined")
except:
  print("Something else went wrong") 
```

---

A bare except is 'not allowed' in pep8. I also do not like it as it makes debugging much harder.

---

:+1: